task_05/imdb.py

```diff
@@ -0,0 +1,68 @@
+"""Read and save data practice."""
+
+import argparse
+
+parser = argparse.ArgumentParser(add_help=True)
+
+parser.add_argument('--year', default=False,
```

Comment:
if you define arguments in this way I should print0` python main.py --year whatever` to print titles with year. You can use `action="store_true"` option instead.
It also concerns --rate and --all keys.
You can find examples in presentation for 5th lesson